File: Patterns/7. Pyramid Patterns/66. Pyramid Even Numbers/pen.py
n = int(input("Enter a Number : "))

for i in range(1,n+1):
    # print spaces
    for j in range(1,n-i+1):
        print(end='   ')
    # print first half
    for j in range(1,i+1):
        print(f"{2*j:3}",end='')
    # print second half
    for j in range(i-1,0,-1):
        print(f"{2*j:3}",end='')
    print()


'''

Output:

              2
           2  4  2
        2  4  6  4  2
     2  4  6  8  6  4  2
  2  4  6  8 10  8  6  4  2

'''

File: Patterns/07. Pyramid Patterns/66. Pyramid Even Numbers/pen.cpp
#include <iostream>
#include <iomanip>
using namespace std;
int main(){
    int n;
    cout<<"Enter a Number : ";
    cin>>n;

    for(int i=1; i<=n; i++){
        // print spaces
        for(int j=1; j<=n-i; j++){
            cout<<"   ";
        }
        // print first half
        for(int j=1; j<=i; j++){
            cout<<setw(3)<<2*j;
        }
        // print second half
        for(int j=i-1; j>=1; j--){
            cout<<setw(3)<<2*j;
        }
        cout<<'\n';
    }

}

/*
Output

              2
           2  4  2
        2  4  6  4  2
     2  4  6  8  6  4  2
  2  4  6  8 10  8  6  4  2

*/